indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -428,7 +428,7 @@ private void manageInternalCritical(
               errorMessage = e.getMessage();
             } else {
               errorMessage = "Failed while waiting for the task to be ready to run. "
-                                          + "See overlord logs for more details.";
+                             + "See overlord logs for more details. Error: " + e.getMessage();
```

Comment:
Something like this would be more user-friendly:

```suggestion
                           errorMessage = StringUtils.format("Encountered error[%s] while waiting for task to be ready. See Overlord logs for more details.", e.getMessage());
```